AdvancedTask/AdvancedTask/Pages/Components/ProfileOverview/ProfileUserDeatilsComponent.cs

```diff
@@ -0,0 +1,116 @@
+﻿using AdvancedTask.Utilities;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using SeleniumExtras.WaitHelpers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AdvancedTask.Test_Model;
+
+namespace AdvancedTask.Pages.Components.ProfileOverview
+{
+    public class ProfileUserDeatilsComponent:BaseClass
+    {
+      
+        private IWebElement SaveButton;
+        private IWebElement AvailabilityDropdown;
+        private IWebElement HoursDropdown;
+        private IWebElement EarnTargetDropdown;
+        private IWebElement PopUpMessage;
+        private string Message = "";
+       
+        public void renderAddMessage()
+        {
+            try
```

Comment:
any reason for try catch block?

Reply:
Yes, it was showing error "Unable to locate element" as it was searching for element before executing the tests.